hoomd/md/ForceComposite.cc

```diff
@@ -973,6 +982,54 @@ void ForceComposite::updateCompositeParticles(uint64_t timestep)
         updated_pos += dr_space;
         quat<Scalar> updated_orientation = orientation * local_orientation;
 
+        // update velocity of constituents
+        // central particle angular momentum, velocity, moment of inertia
+        quat<Scalar> angmom(h_angmom.data[central_idx]);
+        Scalar4 velocity = h_velocity.data[central_idx];
+        vec3<Scalar> vel(velocity);
+        vec3<Scalar> I(h_inertia.data[central_idx]);
+
+        // get mass of constituent
+        const Scalar4 vel_mass = h_velocity.data[particle_index];
+        double mass = vel_mass.w;
```

Comment:
`Scalar` is a typedef to `float` or `double` based on how HOOMD was compiled.
```suggestion
        const Scalar mass = vel_mass.w;
```